tests/unit_tests/dao/dataset_test.py

```diff
@@ -51,7 +51,7 @@ def test_validate_update_uniqueness(session: Session) -> None:
     db.session.add_all([database, dataset1, dataset2])
     db.session.flush()
 
-    # same table name, different schema
```

Comment:
These comments were ordered wrongly and thus misleading. It's probably best to not have comments if the logic is clear from the code.